app/modules/sightings/resources.py

```diff
@@ -472,20 +475,17 @@ def patch(self, args, sighting):
             enc_json_data = []
             try:
                 for arg in edm_args:
-                    if (
-                        arg.get('path', None) == '/encounters'
-                        and arg.get('op', None) == 'add'
+                    if arg.get('path', None) == '/encounters' and (
```

Comment:
@naknomum Can you say if you agree with this change. It jumped out at me as being wrong so I changed it.

Reply:
i do not think we have a "standard" way to treat `op=add` vs `op=replace` when the property/path is a **list**.  so i am fine with this following the same equivalence pattern as single-value properties.   unless @brmscheiner needs a specific call to replace the old list with a new list.

Reply:
oh i see yes i think you are right.  this building of `anns` _should_ be done for either `op`.   good call.